File: Java/Day1Programs/src/PrimeOrNot.java
//8:Check if number is a prime number or not.
import java.util.Scanner;
public class PrimeOrNot {

	public static void main(String[] args) {
		int num;
		boolean flag= false;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter num");
		num = sc.nextInt();
		for(int i = 2; i<= num/2; i++) {
			if(num%i == 0) {
				flag = true;
				System.out.println("Number is not Prime \n");
				break;
			}
		}
		if(flag == false) {
			System.out.println("Number is Prime \n");
		}

	}

}

File: C/C Programs/PrimeOrNot.c
#include<stdio.h>
#include<math.h>
void main()
{
	int num, i, flag = 0;
	printf("enter num \n");
	scanf("%d", &num);
	for(i=2; i<= num/2; i++)
	{
		if(num % i == 0)		
		{
			printf("num is not prime \n");
			flag =1;
			break;
		}
	}
		if(flag == 0)
			printf("Prime Num \n");
}
